Packs/FireEyeHX/Integrations/FireEyeHXv2/FireEyeHXv2.yml

```diff
@@ -1687,21 +2121,265 @@ script:
       description: The share mode of the indicator category.
       type: string
     description: Lists the indicator categories.
+    deprecated: false
+    execution: false
   - name: fireeye-hx-delete-indicator
     arguments:
     - name: category
       required: true
       description: The category name.
+      default: false
+      isArray: false
+      secret: false
     - name: indicator_name
       required: true
       description: The name of the indicator. Use the `uri_name` value.
+      default: false
+      isArray: false
+      secret: false
     description: Delete an indicator.
+    deprecated: false
+    execution: false
+  - arguments:
+    - default: false
+      description: The host set id.
+      isArray: false
+      name: host_set_id
+      required: true
+      secret: false
+    deprecated: false
+    description: Deletes a host set.
+    execution: true
+    name: fireeye-hx-delete-host-set
```

Comment:
let's add for this command the outputs: `FireEyeHX.HostSets._id` and `FireEyeHX.HostSets.deleted`
where deleted is true, of course.
in other commands, we will have this path too, where `deleted` is `false`. (we did it with Sentinel too if you remember)